Sources/Network/NetworkConfig.swift

```diff
@@ -5,20 +5,23 @@
 import Foundation
 
 struct NetworkConfig {
-    static func make(consensusUrl: String, fogUrl: String, attestation: AttestationConfig, transportProtocol: TransportProtocol)
+    static func make(consensusUrlLoadBalancer: RandomUrlLoadBalancer<ConsensusUrl>, fogUrlLoadBalancer: RandomUrlLoadBalancer<FogUrl>, attestation: AttestationConfig, transportProtocol: TransportProtocol)
         -> Result<NetworkConfig, InvalidInputError>
     {
-        ConsensusUrl.make(string: consensusUrl).flatMap { consensusUrl in
-            FogUrl.make(string: fogUrl).map { fogUrl in
-                NetworkConfig(consensusUrl: consensusUrl, fogUrl: fogUrl, attestation: attestation, transportProtocol: transportProtocol)
-            }
-        }
+        return .success(NetworkConfig(consensusUrlLoadBalancer: consensusUrlLoadBalancer, fogUrlLoadBalancer: fogUrlLoadBalancer, attestation: attestation, transportProtocol: transportProtocol))
     }
 
-    let consensusUrl: ConsensusUrl
-    let fogUrl: FogUrl
-
     private let attestation: AttestationConfig
+    private let consensusUrlLoadBalancer: RandomUrlLoadBalancer<ConsensusUrl>
+    private let fogUrlLoadBalancer: RandomUrlLoadBalancer<FogUrl>
+    
+    var fogUrlsDescription: String {
```

Comment:
don't think these should be member variables on this struct if just for debugging.

Reply:
can we move these string constructions to in-line variables at the call-site ?

Reply:
sure - good idea, will update

Reply:
the URLs are local to the load balancers, which are private members of the NetworkConfig - so the URLs are not accessible...  Will ping you in Slack to discuss how to expose the info for logging, if we want to.

Reply:
Updated NetworkConfig to expose fog and consensus urls (to not expose the load balancers themselves), where the MobileCoinClient extension builds the descriptive string for logging...